source/content/overview/io.md

```diff
@@ -0,0 +1,625 @@
+ -   All parameter sections must have the column \"name\". Names should
+     be unique within a parameter section.
+ -   Stochastic variables should have the usual uncertainty columns,
+     e.g. uncertainty type, loc, scale, minimum, maximum.
+ -   Variables which are defined by formulas should have the column
+ -   Parameters can have an \"amount\" column, but if there is also a
+     \"formula\" column, the amount will be overwritten on import as
+     the formula is evaluated.
+ -   Formulas should be for consumption by Python, not Excel; they will
+     be interpreted by `asteval`(https://newville.github.io/asteval/).
+ -   The `Database`, `Project parameters`, and `Database parameters`
+     don\'t have to be in any order or in any particular worksheet.
+     There can be multiple `Project parameters` and
+     `Database parameters` sections, they will be concatenated.
+
+The following data transformations are applied by the `ExcelImporter`:
+
+ -   Numbers are translated from text into actual numbers.
+ -   Tuples, separated in the cell by the `::` string, are
+     reconstructed, i.e. \"this::example\" would become
+     `("this", "example")`.
+ -   The strings \"True\" and \"False\" (by themselves, not as part of
+     a larger string) are transformed to their respective boolean
+     values.
+ -   Fields with the value `(Unknown)` are omitted on a per-row basis.
+
+### Importing LCIA methods from the standard Excel template
+
+Proper implementation of life cycle impact assessment methods requires a complete set of metadata. However, for simple LCIA methods, Brightway
+has a simplified importer for Excel and CSV files: `bw2io.ExcelLCIAImporter` and `bw2io.CSVLCIAImporter`. These function the same as other importer (i.e. you extract data, apply strategies until you are satisfied with the matching, and write the modified data). There are example [Excel workbooks](https://github.com/brightway-lca/brightway2-io/raw/legacy/tests/fixtures/excel/lcia.xlsx) and [CSV files](https://github.com/brightway-lca/brightway2-io/raw/legacy/tests/fixtures/csv/lcia.csv).
+
+Please note the following:
+
+* Excel workbooks should only have one worksheet.
+* Default matching to biosphere flows is based on ``name`and`categories`.
+
+The convention for`categories`is the same as for inventory datasets: subcategories should be separated with `::`, e.g.`natural resource::in ground`. You can also list the main category and subcategory in separate columns, and use a strategy function to combine them. * The call function to instantiate the LCIA importer objects requires`filepath`,`name`(method name as a tuple),`description`, and`unit`.  What to do with unmatched exchanges? If there are unlinked exchanges, you have several options. If you aren't sure what to do yet, you can save a temporary copy (that can be loaded later) using `.write_unlinked("some name")`. Calling `.statistics()` will show what kind of exchanges aren't linked, e.g.:
+
+```python
+> sp.statistics()
```

Comment:
Would be cool to just use [`sphinx-prompt`](https://sphinx-extensions.readthedocs.io/en/latest/sphinx-prompt.html) to avoid uses copying the `>` characters. I will add it in the version bump to Sphinx 8 later tonight.

Reply:
Sounds helpful. When I copy [the example](https://sphinx-extensions.readthedocs.io/en/latest/sphinx-prompt.html), though, I get:

```console
span.prompt1:before {
  content: ">>> ";
}
import datetime
datetime.now()
datetime.datetime.now()
datetime.datetime(2020, 3, 29, 16, 13, 30, 100129)
```

Which is better but not what I expected?

Reply:
...ok, I may need to check https://github.com/sbrunner/sphinx-prompt/issues/107